set.mm

```diff
@@ -102838,19 +102838,67 @@ Because we use a disjoint union for cardinal addition (as explained in the
     ( ccrd cdm wcel wa cdju cun cdom wbr djunum undjudom numdom syl2anc ) ACDZE
     BOEFABGZOEABHZPIJQOEABKABOOLPQMN $.
 
-  $( The cardinal and ordinal sums of finite ordinals are equal.  (Contributed
-     by Paul Chapman, 11-Apr-2009.)  (Revised by Mario Carneiro,
-     6-Feb-2013.) $)
-  nnadju $p |- ( ( A e. _om /\ B e. _om ) ->
+  ${
+    $d B x $.  $d A x y $.
+    $( The cardinal and ordinal sums of finite ordinals are equal.  For a
+       shorter proof using ~ ax-rep , see ~ nnadjuALT .  (Contributed by Paul
```

Comment:
Since the idea of this `nnadju` proof is to avoid ax-rep , I'd add

```
$( $j usage 'nnadju' avoids 'ax-rep'; $)
```

which will make the automated checks fail if the proof starts using ax-rep for whatever reason. Looks like we have been putting this after the proof but before the `$}`

Reply:
I didn't know that was a thing, thanks!